common/djangoapps/util/tests/test_course.py

```diff
@@ -0,0 +1,48 @@
+""" Tests for course utils. """
```

Comment:
@wajeeha-khalid can we please move the docstrings to new three-line format in this file ?